app/scripts/models/user.js

```diff
@@ -23,6 +23,8 @@ define(function (require, exports, module) {
   const SearchParamMixin = require('models/mixins/search-param');
   const Storage = require('lib/storage');
   const vat = require('lib/vat');
+  const MarketingEmailClient = require('lib/marketing-email-client');
+  const ProfileClient = require('lib/profile-client');
 
   var User = Backbone.Model.extend({
     initialize (options = {}) {
```

Comment:
`marketingEmailClient, profileClient` are currently passed in as `options`, but you just moved those two things into this file. Now in this file those two things need to be initialized. So you need to modify, 

```
this._marketingEmailClient = options.marketingEmailClient;
``` 

and 

```
this._profileClient = options.profileClient;
``` 

below

Reply:
Instead take the code that you added below and do something like:

```
      this._profileClient = new ProfileClient({
        profileUrl: this._config.profileUrl
      });
```

and a similar thing for `marketingEmailClient`.

One thing though, you don't have access to `this._config` on this file. so you need to pass it in as an option to user.js

Reply:
So, after initialising it that way, I need to remove those two functions , namely ````initializeProfileClient```` and ````initializeMarketingClient```` , right?

Reply:
@kritisingh1 Correct!

Reply:
@kritisingh1 You can do most of it inside the `initialize` function (of user.js)